fontscan/fontmap.go

```diff
@@ -334,6 +334,22 @@ func (fm *FontMap) FindSystemFont(family string) (Location, bool) {
 	return Location{}, false
 }
 
+// MatchedSystemFonts return all matched fonts
+func (fm *FontMap) MatchedSystemFonts(family string) []Location {
```

Comment:
Since the existing `FindSystemFont` function is very similar, I would prefer naming it `FindSystemFonts`.